unit/util/piped_process.cpp

```diff
@@ -0,0 +1,98 @@
+/// \file
+/// \author Diffblue Ltd.
+/// Unit tests for checking the piped process communication mechanism.
+
+#ifdef _WIN32
+// No unit tests yet!
+#else
+
+#  include <testing-utils/use_catch.h>
+#  include <unistd.h>
+#  include <util/piped_process.h>
+
+TEST_CASE(
+  "We create a pipe and we can read from it",
+  "[core][util][piped_process]")
+{
+  std::string to_be_echoed = "The Jabberwocky";
+  // Need to give up to avoid shell built-in invocation
+  std::string binary = "/bin/echo";
+  std::string command = binary + " " + to_be_echoed;
+  piped_processt process = piped_processt(command);
+
+  sleep(1);
```

Comment:
???

Reply:
This was temporary while we were iterating to stabilise the tests.

All of the instances of `sleep` have been removed from the current iteration of the test code.